cli/data/get.go

```diff
@@ -135,21 +138,38 @@ func copy(session *session.Session, source S3Path, destination string) {
 	}
 }
 
-func copyToLocalFiles(s3Client *s3.S3, objects []*s3.Object, source S3Path, destination string) {
+func copyToLocalFiles(s3Client *s3.S3, objects []*s3.Object, source S3Path, destination string, files []string) {
 	var (
-		wg  = new(sync.WaitGroup)
-		sem = make(chan struct{}, s3ParallelGets)
+		wg           = new(sync.WaitGroup)
+		sem          = make(chan struct{}, s3ParallelGets)
+		downloadList = filterObjects(objects, files)
```

Comment:
To me reads a bit odd that we use the word "files" everywhere but then these are more like "filepaths" or "s3Keys"

Reply:
IMO `files` are like a whitelist with filenames (or `s3Keys). When I read this piece of code, it sounds to me like: Hey! Filter this list of objects given this list of files that I want to download.



I guess it makes sense to call it `files` given that the flag name is `files`:

```
paddle data get --files file1.csv,file2.csv trained-model/version1 dest/path
```

But, tbh I don't have strong feelings. If we are to change the variable name, I'd also change the flag name though.